clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_LIB_CIRBASEBUILDER_H
```

Comment:
this should be `LLVM_CLANG_CIR_DIALECT_BUILDER_CIRBASEBUILDER_H`

Reply:
I copied the include guard macro names from the same files in the ClangIR incubator and didn't give them any thought.  I'll fix these.  Now that I know what the pattern should be, I'll pay attention to those in future changes.

Reply:
Yup, our fault originally :p